src/System.Private.ServiceModel/tools/setupfiles/BuildWCFSelfHostedService.cmd

```diff
@@ -0,0 +1,49 @@
+@echo off
+setlocal
+
+if not defined VisualStudioVersion (
+    if defined VS140COMNTOOLS (
+        call "%VS140COMNTOOLS%\VsDevCmd.bat"
+        goto :EnvSet
+    )
+
+    if defined VS120COMNTOOLS (
+        call "%VS120COMNTOOLS%\VsDevCmd.bat"
+        goto :EnvSet
+    )
+
+    echo Error: %~nx0 requires Visual Studio 2013 or 2015.
+    echo        Please see https://github.com/dotnet/wcf/wiki/Developer-Guide for build instructions.
```

Comment:
... for later, perhaps you could update this link from wherever you copied from too :) 
Not blocking, we can take the change later